earthaccess/results.py

```diff
@@ -111,14 +110,15 @@ def get_umm(self, umm_field: str) -> Union[str, Dict[str, Any]]:
     def concept_id(self) -> str:
         """
         Returns:
-          Retrurns a collection's `concept_id`, this id is the most relevant search field on granule queries.
+            A collection's `concept_id`.
+            This id is the most relevant search field on granule queries.
```

Comment:
Hey @danielfromearth, thanks for the additional cleanup :) I have some more details in the PR description, but indentation on multi-line strings is super important for the docs to render correctly! I can't comment a suggestion on my own PR :roll_eyes: , but should look like:

```
Returns:
    A collection's `concept_id`.
        This id is the most relevant search field on granule queries
```

Without the additional indentation, it will render two rows of return documentation.

Reply:
That totally works for me. 
For clarity though, let me ask further... is the additional indentation a ReadTheDocs thing?  
The second example in the [Google style guide (section 3.8.3)](https://google.github.io/styleguide/pyguide.html#383-functions-and-methods) shows the `Args:` section with no additional indentation. Here is the relevant docstring example copied for convenience:

```
Args:
      table_handle:
        An open smalltable.Table instance.
      keys:
        A sequence of strings representing the key of each table row to
        fetch.  String keys will be UTF-8 encoded.
      require_all_keys:
        If True only rows with values set for all keys will be returned.
```

(Note that `earthaccess` currently uses `Parameters:` throughout its codebase instead of `Args:`.)

Reply:
I'm not 100% sure. I struggled with the docstring style a lot. I was certain I had made a fix exactly like this in several other places in this file to fix rendering, but those fixes might have been subtly different than what I'm commenting on now. I've replaced a lot of that context with other stuff, I just remember checking over the output exhaustively and feeling confident about it before setting it down. I trust you're doing similarly exhaustive checking on your end, so if it's working I'm happy :)

Reply:
I'm adding the indentation, since it seems like it is necessary for the readthedocs to display correctly.

Reply:
Weird. It feels like we could/should have better validation for docstrings, but haven't taken the time to explore that yet.

Another thing I'm not clear on is why we're using `Parameters:` sections instead of `Args:`. Since it's working, I just decided to focus on other things :)